### ISSUE TITLE
Wrap Content not working if Caldroid isn't the first child in ScrollView (Samsung S6 Edge)

### ISSUE THREAD

**Issue by rohannexialabs** [roomorama/Caldroid#314]
I have seen some strange issue, I have Caldroid inside a ScrollView. When I make Caldroid as the first child then "wrap_content" is working fine but if I make Caldroid as second child, the dates in the calendar are not visible, just month and weeks are visible.

Note that this is occurring on some devices only such as Samsung S6 Edge. I tested it on emulator and my personal device (YU Yureka), on which it was working fine. If I make the "Switch" as "gone" then "wrap_content" is working fine.

My Layout - 
```
<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:fillViewport="true">

    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:custom="http://schemas.android.com/apk/res-auto"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:background="@drawable/repeat_background_login"
        android:orientation="vertical">

        <Switch
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:id="@+id/sStatus"
            android:text="@string/publish_property"
            android:textAllCaps="false"
            android:textColor="@color/white"
            android:textSize="@dimen/large_text"
            android:layout_margin="20dp"
            custom:fontName="RobotoSlab_Bold"
            android:visibility="visible"/>

        <LinearLayout
            android:id="@+id/calendar1"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:layout_marginTop="20dp"
            android:orientation="vertical" />
         ...
```
The present workaround which I could find was https://github.com/roomorama/Caldroid/issues/273 which is working for all devices but I don't want to put it like that.

**Comment by thomasdao** [comment]
ScrollView within a scrollView is not very well supported by Android - sorry that I cannot help you much. Let's hope that Google does better job in handling these cases

**Comment by thomasdao** [comment]
@rohannexialabs I have fixed this issue in latest Caldroid 2.3.0, let me know if it works, thanks
